### PR TITLE
[EN] Skinning/Interface: Redo images; Simplify tables; Use captions instead of headings

### PR DESCRIPTION
**Pull request by MegaApplePi**
- interface
  - remove detailed info about skinning element (tables)
  - fix all skinning element notes
  - redo images (downscale and optimize)
  - make use of captions instead of headings
  - much better \\:D/

### CODE REVIEW

**Comment by Deimosu** [comment]
no skin element names? or do you really expect from the reader to hover above the image (even unseen images due to its color) to get the name

**Comment by MegaApplePi** [comment]
@deimosu
> make use of captions instead of headings

File: wiki/Skinning/Interface/en.md [hunk 1]
```diff
@@ -1,1804 +1,1366 @@
 [true]: /wiki/shared/True.png
 [false]: /wiki/shared/False.png
 
-# Skinning Interface
+# Interface
 
-## Welcome Screen
+_See also: [Skinning Interface Tutorial](/wiki/Skinning_Interface_Tutorial) and [Interface](/wiki/Interface)_
```
Comment (TPGPL) [inline wiki/Skinning/Interface/en.md]: Also see?

File: wiki/Skinning/Interface/en.md [hunk 2]
```diff
@@ -1,1804 +1,1366 @@
...
 
-### menu-background.jpg
+Interface skinning elements are used in multiple game modes or part of the osu!client's user interface.
```
Comment (TPGPL) [inline wiki/Skinning/Interface/en.md]: or parts of

File: wiki/Skinning/Interface/en.md (hunk 1 above)
Comment (MegaApplePi) [inline wiki/Skinning/Interface/en.md]: ¯\\\_(ツ)\_/¯ Wikipedia uses `_See also: [...]_`

**Pull request merged**